amazonka-s3-streaming.cabal

```diff
@@ -22,9 +22,9 @@ library
   default-language:    Haskell2010
   ghc-options:         -Wall
   build-depends:       base >= 4.9 && < 5
-                       , amazonka         >= 1.6.1      && < 1.7
-                       , amazonka-core    >= 1.6        && < 1.7
-                       , amazonka-s3      >= 1.6        && < 1.7
+                       , amazonka         >= 2.0        && < 2.1
+                       , amazonka-core    >= 2.0        && < 2.1
```

Comment:
You shouldn't need to depend on core.

Reply:
```
src/Network/AWS/S3/StreamingUpload.hs:27:1: error:
    Could not load module ‘Amazonka.Prelude’
    It is a member of the hidden package ‘amazonka-core-2.0’.
    Perhaps you need to add ‘amazonka-core’ to the build-depends in your .cabal file.
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
27 | import Amazonka.Prelude (MonadResource)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Network/AWS/S3/StreamingUpload.hs:29:1: error:
    Could not load module ‘Amazonka.Data.Body’
    It is a member of the hidden package ‘amazonka-core-2.0’.
    Perhaps you need to add ‘amazonka-core’ to the build-depends in your .cabal file.
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
29 | import Amazonka.Data.Body   ( HashedBody(..) )
```
Is there a better import for these?

Reply:
get MonadResource from `Control.Monad.Trans.Resource` (`resourcet`) and `HashedBody` from `Amazonka`.